addons/purchase/tests/test_access_rights.py

```diff
@@ -43,12 +53,36 @@ def setUpClass(cls, chart_template_ref=None):
             'property_account_expense_categ_id': cls.account_expense_product.id
         })
         cls.product = cls.env['product.product'].create({
+            'company_id': cls.company.id,
             'name': "Product",
             'standard_price': 200.0,
             'list_price': 180.0,
             'type': 'service',
         })
 
+        cls.purchase_order_1 = cls.env['purchase.order'].create({
+            "company_id": cls.company.id,
+            "partner_id": cls.vendor.id,
+            "user_id": user_admin.id,
+            "state": "draft",
+            "order_line": [(5, 0, 0),
```

Comment:
State's `draft` by default, and since it's a new record, the 'clear' command seems useless
```suggestion
            "order_line": [
```